### ISSUE TITLE
Compilation failure with elixir-1.13.0-rc.0

### ISSUE THREAD

**Issue by simoncocking** [tompave/fun_with_flags#113]
`fun_with_flags` 1.6.0 fails to compile under `elixir-1.13.0-rc.0`, with the following error:

```
$ mix deps.compile fun_with_flags
==> fun_with_flags
Compiling 7 files (.ex)

== Compilation error in file lib/fun_with_flags/protocols/actor.ex ==
** (CompileError) lib/fun_with_flags/protocols/actor.ex:116: undefined function defdelegate/2 (there is no such import)
```

Per the Elixir 1.13.0 [release notes](https://github.com/elixir-lang/elixir/releases/tag/v1.13.0-rc.0):
> [Protocol] Add defdelegate to the list of unallowed macros inside protocols as protocols do not allow function definitions

**Comment by tompave** [comment]
Hello, thank you for reporting the issue.
I believe that this is fixed in `v1.7.0`, just released. To be honest it was fixed on master a while ago, but I forgot to cut a release. 😬 

Please give it a try and let me know! :-)

**Comment by simoncocking** [comment]
Sorry for the late reply, but yeah `1.7.0` is all good 👍
